### PR TITLE
Initial draft of release notes for August preview

### PR DESCRIPTION
**Pull request by adrianhall**
Replaces PR #542

### CODE REVIEW

File: _posts/2019-08-06-java-preview2.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+title: Azure SDK for Java (August 2019 Preview)
+layout: post
+tags: java
+sidebar: java_sidebar
+repository: azure/azure-sdk-for-java
+---
+
+The Azure SDK team is pleased to make available the August 2019 client library preview release. This represents the first release of the ground-up rewrite of the client libraries to ensure consistency, idiomatic design, and excellent developer experience and productivity. This preview release includes new client libraries for Azure App Configuration, Cosmos, Identity, Key Vault (keys and secrets), Event Hubs, and Storage (blob).
```
Comment (joshfree) [inline _posts/2019-08-06-java-preview2.md]: should be `Storage`, unqualified or `Storage (blob, files, queues)`

File: _posts/2019-08-06-java-preview2.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+## Changelog
+Detailed change logs are linked to in the Quick Links below. Here are some critical call outs.
+
+* Designed based on the [Azure SDK Design Guidelines for Java][java-guidelines], resulting in a consistent API design and common feature set such as HTTP retries, logging, transport protocols, authentication protocols, etc.
+* Modernized API making use of Java 8 features such as streams, new date / time, functional interfaces, etc, to offer an improved and more productive developer experience.
+* Synchronous and asynchronous APIs offer developers simplicity for simple cases, and full asynchronousity when performance and scalability matters. Reactive streams are offered using [Project Reactor](http://projectreactor.io).
+
+### Azure Identity
+
+- Added new user credential types, allowing authentication within client applications.
```
Comment (joshfree) [inline _posts/2019-08-06-java-preview2.md]: Added new user credential types, allowing authentication flows (Interactive, Device code, Username/password).  See [Microsoft Authentication Library (MSAL) authentication flows](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-authentication-flows) for more details.

File: _posts/2019-08-06-java-preview2.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+| Key Vault - Secrets | [Readme](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-keyvault-secrets/README.md) | [Changelog](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-keyvault-secrets/CHANGELOG.md) | [Samples](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/keyvault/azure-keyvault-secrets/src/samples/java) | [JavaDoc](https://azure.github.io/azure-sdk-for-java/track2reports/index.html) |
+| Event Hubs | [Readme](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/eventhubs/README.md) | [Changelog](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/eventhubs/CHANGELOG.md) | [Samples](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/eventhubs/azure-eventhubs/src/samples/java) | [JavaDoc](https://azure.github.io/azure-sdk-for-java/track2reports/index.html) |
+| Storage - Blob | [Readme](https://github.com/Azure/azure-sdk-for-java/blob/master/storage/client/blob/README.md) | [Changelog](https://github.com/Azure/azure-sdk-for-java/blob/master/storage/client/blob/CHANGELOG.md) | - | [JavaDoc](https://azure.github.io/azure-sdk-for-java/track2reports/index.html) |
```
Comment (joshfree) [inline _posts/2019-08-06-java-preview2.md]: is it possible to link to something more specific to each individual service for the JavaDoc links?

File: _posts/2019-08-06-java-preview2.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  <groupId>com.azure</groupId>
+  <artifactId>azure-storage-queue</artifactId>
+  <version>12.0.0-preview.2</version>
+</dependency>
+```
+
+## Feedback
+If you have a bug or feature request for one of the libraries, please post an issue to [GitHub](https://github.com/azure/azure-sdk-for-java/issues).
+
+## Changelog
```
Comment (joshfree) [inline _posts/2019-08-06-java-preview2.md]: could you add distributed tracing to the changelog and link to the tracing samples that @loarabia is publishing?

File: _posts/2019-08-06-js-preview2.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+| [Storage - Queues](https://docs.microsoft.com/en-us/azure/storage/queues/storage-dotnet-how-to-use-queues) | [Package](https://www.npmjs.com/package/@azure/storage-queue/v/12.0.0-preview.2) | [Readme.md](https://github.com/Azure/azure-sdk-for-js/tree/feature/storage/sdk/storage/storage-queue) | [API Reference Documentation](https://azure.github.io/azure-sdk-for-js/storage-queue/index.html) | [Changelog](https://github.com/Azure/azure-sdk-for-js/releases/tag/%40azure%2Fstorage-queue_12.0.0-preview.2) | [Samples](https://github.com/Azure/azure-sdk-for-js/tree/feature/storage/sdk/storage/storage-queue/samples) |
+| [Event Hubs](https://azure.microsoft.com/en-us/services/event-hubs/) | [Package](https://www.npmjs.com/package/@azure/event-hubs/v/5.0.0-preview.1\2) | [Readme.md](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-hubs) | [API Reference Documentation](https://azure.github.io/azure-sdk-for-js/event-hubs/index.html) | [Changelog](https://github.com/Azure/azure-sdk-for-js/releases/tag/%40azure%2Fevent-hubs_5.0.0-preview.2) | [Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-hubs/samples) |
```
Comment (ramya-rao-a) [inline _posts/2019-08-06-js-preview2.md]:
```suggestion
| [Event Hubs](https://azure.microsoft.com/en-us/services/event-hubs/) | [Package](https://www.npmjs.com/package/@azure/event-hubs/v/5.0.0-preview.2) | [Readme.md](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-hubs) | [API Reference Documentation](https://azure.github.io/azure-sdk-for-js/event-hubs/index.html) | [Changelog](https://github.com/Azure/azure-sdk-for-js/releases/tag/%40azure%2Fevent-hubs_5.0.0-preview.2) | [Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-hubs/samples) |
```

File: _posts/2019-08-06-js-preview2.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+- Support for Azure Active Directory credentials created using the new [@azure/identity](https://www.npmjs.com/package/@azure/identity) library. Moving forward, this would be the common mode of authentication for all Azure SDK libraries.
+- Modernized API making use of async functions and async iterators, to offer an improved and more productive developer experience.
+
+### Azure Identity
+
+- Added new user credential types, allowing authentication within client applications.
+
+### Event Hubs
+
+- Added batch send capabilities.
```
Comment (ramya-rao-a) [inline _posts/2019-08-06-js-preview2.md]:
```suggestion
- Expanded support for sending multiple messages in a single call by adding the ability to create a batch which avoids the error scenario of exceeding size limits. Users having bandwidth concerns can control the batch size as desired.
```

Comment (ramya-rao-a) [reply _posts/2019-08-06-js-preview2.md]: Same for other languages

File: _posts/2019-08-06-js-preview2.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+- Added a new EventProcessor for receiving events as an early preview.
```
Comment (ramya-rao-a) [inline _posts/2019-08-06-js-preview2.md]:
```suggestion
- Introduced a new model for consuming events via the EventProcessor class. This simplifies the process of checkpointing today and will handle load balancing across partitions in upcoming updates.
```

Comment (ramya-rao-a) [reply _posts/2019-08-06-js-preview2.md]: Same for other languages

File: _posts/2019-08-06-java-preview2.md (hunk 3 above)
Comment (adrianhall) [reply _posts/2019-08-06-java-preview2.md]: I don't think so - it's based on JavaDoc.  If you have a more specific URL, let me know and it's easy to update.

File: _posts/2019-08-06-js-preview2.md (hunk 7 above)
Comment (adrianhall) [reply _posts/2019-08-06-js-preview2.md]: I dislike this wording.  For one, it's a forward-looking statement.  It also loses the point that its an early preview for feedback.  I've made the change, but we probably want to think about this one a bit more.

**Pull request merged**